components/brave_new_tab_ui/components/default/braveToday/customize/SourcesListEntry.tsx

```diff
@@ -59,26 +69,32 @@ function FavIcon (props: { src?: string }) {
 
 export function FeedListEntry (props: Props) {
   const publisher = usePublisher(props.publisherId)
-  const { subscribed, setSubscribed } = usePublisherSubscribed(props.publisherId)
+  const { followed, setFollowed } = usePublisherFollowed(props.publisherId)
 
-  return <Container direction="row" justify="space-between" align='center' onClick={() => setSubscribed(!subscribed)}>
+  return <Container direction="row" justify="space-between" align='center'>
     <Flex align='center' gap={8}>
       <FavIcon src={publisher.faviconUrl?.url} />
       <Text>{publisher.publisherName}</Text>
     </Flex>
-    <ToggleButton>
-      {subscribed ? Heart : HeartOutline}
+    <ToggleButton onClick={() => setFollowed(!followed)}>
+      {followed
+        ? getLocale('braveNewsFollowButtonFollowing')
```

Comment:
I don't know if this button is ever in a state where the source is not being followed? Non-followed sources are not shown in the list.

Reply:
ok, I removed the other state